includes/base/element-base.php

```diff
@@ -430,6 +444,11 @@ public function add_link_attributes( $element, array $url_control, $overwrite =
 	public function print_element() {
 		$element_type = $this->get_type();
 
+		if ( $this->should_render_shortcode() ) {
+			echo '[elementor-element data="' . esc_attr( base64_encode( wp_json_encode( $this->get_raw_data() ) ) ) . '"]';
```

Comment:
@KingYes Can you please check if you need to use `do_shortcode()` here? I see when the content is fetched via the REST API it renders `[elementor-element data="..."]`, instead of the actual HTML.